server/cloudapps/proxy.go

```diff
@@ -60,14 +84,19 @@ func (p *proxy) SendChangeNotification(s *Subscription, msg interface{}) {
 		}
 	}()
 
-	u := path.Join(app.RootURL, "notify", string(SubjectUserJoinedChannel))
-	// <><> TODO ticket: progressive backoff on errors
-	resp, err := client.Post(u, "application/json", piper)
+	req, err := http.NewRequest(http.MethodPost, path.Join(app.Manifest.RootURL, "notify", string(SubjectUserJoinedChannel)), piper)
+	if err != nil {
+		// <><> TODO log
+		return
+	}
+	req.Header.Set("Authorization", "Bearer "+token)
+	resp, err := client.Do(req)
 	if err != nil {
 		// <><> TODO log
+		// TODO ticket: progressive backoff on errors
 		return
 	}
-	defer resp.Body.Close()
+	resp.Body.Close()
```

Comment:
Is there any reason to keep the body open in any branch (mainly the error handling)? If not, we can use the defer at the end of the function.

Reply:
Later if we need to... KISS for now. This came up before when the error messages were of unanticipated types, better NOT happen to us here!